src/bridge/transactions/pre_signed_musig2.rs

```diff
@@ -67,7 +70,8 @@ pub fn push_nonce<T: PreSignedTransaction + PreSignedMusig2Transaction>(
 }
 
 pub fn get_nonce_message(nonce: &PubNonce) -> Message {
-    Message::from_hashed_data::<bitcoin::hashes::sha256::Hash>(nonce.to_bytes().as_slice())
+    let nonce_hash = sha256::Hash::hash(nonce.to_bytes().as_slice());
+    Message::from_digest_slice(nonce_hash.as_ref()).expect("Failed to create nonce message")
```

Comment:
I decide to `.expect()` here to not break the API by changing the function signature to return a `Result`.